code/src/playlist.css

```diff
@@ -0,0 +1,46 @@
+.playlist-container {
```

Comment:
This file would need some cleaning up to do. Work on the indention, spacing and the lines :)